File: api/src/main/java/com/hirrao/javaee/service/UserIntroService.java
package com.hirrao.javaee.service;

import com.hirrao.javaee.entity.UserIntro;
import org.springframework.stereotype.Service;

@Service
public interface UserIntroService {
    UserIntro findUserIntro(Long uid);
    void updateUserIntro(Long uid, String intro);
    void setUserIntro(Long uid, String intro);
}

File: api/src/main/kotlin/com/hirrao/javaee/service/UserIntroService.kt
package com.hirrao.javaee.service

import com.hirrao.javaee.entity.UserIntro
import org.springframework.stereotype.Service

@Service
interface UserIntroService {
    fun findUserIntro(uid: Int): UserIntro?
    fun updateUserIntro(uid: Long, intro: String?)
    fun setUserIntro(uid: Int, intro: String?)
}
